Nosach/web-quiz-project/src/main/java/config/QuizConfiguration.java

```diff
@@ -18,22 +22,19 @@
     public SpringLiquibase springLiquibase(){
         SpringLiquibase springLiquibase = new SpringLiquibase();
         springLiquibase.setDataSource(dataSource());
-        springLiquibase.setChangeLog("classpath:changelog-db.xml");
+        springLiquibase.setChangeLog("/resources/changelog-db.xml");
         springLiquibase.setContexts("test, production");
         return springLiquibase;
     }
 
     @Bean
-    public BasicDataSource dataSource(){
+    public DataSource dataSource(){
         BasicDataSource dataSource = new BasicDataSource();
         dataSource.setDriverClassName("com.mysql.jdbc.Driver");
-        dataSource.setUrl("jdbc:mysql://localhost:3306/quiz-project?useUnicode=true" +
-                                                                    "&characterEncoding=UTF8" +
-                                                                    "&useJDBCCompliantTimezoneShift=true" +
-                                                                    "&useLegacyDatetimeCode=false" +
-                                                                    "&serverTimezone=UTC");
-        dataSource.setUsername("root");
-        dataSource.setPassword("root");
+        dataSource.setUrl("jdbc:mysql://lt80glfe2gj8p5n2.chr7pe7iynqr.eu-west-1.rds.amazonaws.com:3306/vt8gqxfgfiqppxq6?reconnect=true");
```

Comment:
Commiting your urls with credentials to public repository is not good idea

Reply:
Ok, will fix this in next commit